nextflow.config

```diff
@@ -149,6 +154,20 @@ params {
     motus_remove_ncbi_ids           = false
     motus_save_mgc_read_counts      = false
 
+    // kmcp
+    run_kmcp               = false
+    kmcp_mode              = 3      // default  kmcp profiling value
```

Comment:
Should this be `0` as in tests?

Reply:
No, default is fine for normally data I think! Again test data is not 'realistic' so don't need such a sensitive value

Reply:
Sounds good.